pkg/antctl/raw/multicluster/deploy/deploy_helper.go

```diff
@@ -170,19 +174,17 @@ func deploy(cmd *cobra.Command, role string, version string, namespace string, f
 			}
 
 			content := string(b)
-			if role == leaderRole && strings.Contains(manifest, "namespaced") {
+			if role == leaderRole && strings.Contains(manifest, "namespaced") && namespace != "antrea-multicluster" {
```

Comment:
Define a constants for "antrea-multicluster" and "kube-system" (I remember there were two constants defined already).

Reply:
Done